### DISCUSSION TITLE
if laravel-backup support conception dB on another server

### DISCUSSION THREAD

**Discussion by hosamalzagh** [spatie/laravel-backup#1439] [Q&A]
hello i  have 2server  
first have files Laravel  8
second have  db

when i run  `php artisan backup:run`

i get this error  

`Backup failed because The dump process failed with exitcode 2 : Misuse of shell builtins : mysqldump: Got error: 1044: "Access denied for user ''USERNAME@'10.0.0.2' to database 'DATABASE NAME'" when using LOCK TABLES
`

**Comment by PaolaRuby** [comment]
Your user do not have privilegies for dump
